src/Topologies/Topologies.jl

```diff
@@ -43,9 +44,9 @@ abstract type AbstractDistributedTopology <: AbstractTopology end
 coordinate_type(topology::AbstractTopology) = coordinate_type(domain(topology))
 
 """
-    domain(topology)
+    Topologies.domain(topology::AbstractTopology)
```

Comment:
```suggestion
    domain(topology::AbstractTopology)
```

Reply:
This was again my attempt at resolving the error (see above [comment](https://github.com/CliMA/ClimaCore.jl/pull/1035#issuecomment-1341645780)). I found some threads that discuss this (e.g., see [this](https://discourse.julialang.org/t/calling-docs-twice-for-the-same-function-in-documenter-jl/37791))

Reply:
@charleskawczynski , I have applied the suggestion, but the problem is still present (see my comment in the PR convo)

Reply:
Ah, I see, we're importing the function and then re-defining the function stub and documenting that one. That actually seems a bit odd to me, @simonbyrne, should we just remove the duplicate function stub or leave it and document both qualified functions (e.g., `Topologies.domain(topology::AbstractTopology)`).

I suppose it's benign, so I'm fine with either one

Reply:
I ended up removing the duplicated stub docstring. There was no way that by qualifying it, Documenter would be happy with it.